charts/fluent-bit/values.yaml

```diff
@@ -11,6 +11,11 @@ image:
   pullPolicy: Always
   # tag:
 
+testImage:
+  repository: busybox
+  pullPolicy: Always
+  tag: latest
```

Comment:
```suggestion
testFramework:
  image: busybox
  pullPolicy: Always
  tag: latest
```

nit: Kindly nest in `testFramework` as this is a pattern seen in other charts and allows for future test related config to be neatly added

Reply:
repository was used to mirror what is already used in this chart, but if want different names that's not a problem

Reply:
yeah that is a good point, though I'd err on the side of the already somewhat common pattern: https://github.com/helm/charts/search?q=testframework

Reply:
Ugh maybe I was too hasty there, the pattern seen in the legacy charts repo is not perfect... what about something like:

```yaml
test:
  image:
    repository: busybox
```

Reply:
haha, sure - will there only ever be one test image or do you want a key for each potential test :D
```
test:
  connection:
    image:
      repository: busybox
  someothertest:
    image:
      repository: bats
```